packages/apps-config/src/endpoints/testing.ts

```diff
@@ -22,7 +22,7 @@ export function createTesting (t: TFunction, firstOnly: boolean, withSort: boole
       isDevelopment: false,
       text: t('rpc.matic', 'Polygon', { ns: 'apps-config' }),
       providers: {
-        MATIC: 'wss://devnet-avail.polygon.technology/ws'
+        MATIC: 'wss://testnet.polygonavail.net/ws'
```

Comment:
configurable through ENV?

Reply:
yeh the other PR does that! and will make change to take a default one!